schedulers/terraform/managed-airflow-mwaa/main.tf

```diff
@@ -3,8 +3,8 @@
 #------------------------------------------------------------------------
 
 module "mwaa" {
-  source  = "aws-ia/mwaa/aws"
-  version = "0.0.1"
+  source = "github.com/askulkarni2/terraform-aws-mwaa?ref=fix%2Fprovider-versions"
```

Comment:
Shall we hold on to the PR until the versions fix is applied to `aws-ia/mwaa/aws` module and update this source and version?

Reply:
Yeah we should hold. That updated MWAA PR is up.

Reply:
I have published a new tag for the recent merge on MWAA. You can use `0.0.2` and update the PR.